src/component/Header/Header.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
It is not recommended to disable eslint rules. Remove this line and add an appropriate label to the button on line 27. You can use aria-label attributes for this